src/pages/core/external-plugins/app-data.md

```diff
@@ -163,15 +165,15 @@ pub async fn add_attributes_plugin() {
 
     let last_blockhash = rpc_client.get_latest_blockhash().await.unwrap();
 
-    let add_attribute_plugin_tx = Transaction::new_signed_with_payer(
+    let add_add_data_plugin_tx = Transaction::new_signed_with_payer(
```

Comment:
nit: Should this be add_app_data_plugin?

Reply:
yes it should. I've changed this at least 3 times though. Dunno why it's not commiting in. Let me try few more times.